patronload/cli.py

```diff
@@ -33,8 +35,8 @@ def main() -> None:
     config_values = load_config_values()
     root_logger = logging.getLogger()
     logger.info(configure_logger(root_logger, os.getenv("LOG_LEVEL", "INFO")))
+    stream = create_log_stream_for_email(root_logger)
```

Comment:
question just so I can understand better... Creating this log stream doesn't prevent logs from going to cloud watch, it just adds an additional place where log messages end up, is that right?

Reply:
Correct, this is an additional handler and everything still outputs to CloudWatch as expected as you'll see when running it manually